### ISSUE TITLE
enet_weights_zoo/cityscapes_weights.sh: weights2.caffemodel is not a zip file

### ISSUE THREAD

**Issue by ckhung** [TimoSaemann/ENet#38]
Hi, when executing `sh enet_weights_zoo/cityscapes_weights.sh` it gives the error message `... unzip:  cannot find zipfile directory in one of /ENet/enet_weights_zoo/weights2.caffemodel ...` The file does not seem to be a zip file.

**Comment by npv0** [comment]
Same issue  @ckhung is it working with f[ix weight downloader](https://github.com/ckhung/ENet/commit/7dc74658c27a3ff85516f7fbb33b2b362957348a) ?

**Comment by ckhung** [comment]
Yes. Simply download weights2.caffemodel and do not process it any further. Do not unzip, do not rm. I did a few other changes according to my personal preference and requirement, therefore didn't create a pull request.

**Comment by thundertrick** [comment]
Finally Make it.
The 2 models can and should be unziped, or u may get a check failed error.

After unzip the models (not rm), my zoo folder is like this:
```
../enet_weights_zoo
├── README.md
├── cityscapes_weights.caffemodel
├── cityscapes_weights.sh
├── cityscapes_weights_before_bn_merge.caffemodel
├── weights.caffemodel
└── weights2.caffemodel
```
And `cityscapes_weights.caffemodel` is the model in tutorial.

---

In @ckhung 's case, I guess the problem is the path of model: `/ENet/enet_weights_zoo/weights2.caffemodel`, which should be `enet_weights_zoo/weights2.caffemodel`.
The shell script might get a wrong directory.
I download and unzip the model manually and everything works well.
